requirements/edx/base.txt

```diff
@@ -6,7 +6,7 @@
 #
 -e git+https://github.com/edx/acid-block.git@98aecba94ecbfa934e2d00262741c0ea9f557fc9#egg=acid-xblock
 -e common/lib/capa
--e git+https://github.com/edx/codejail.git@33758da2609bd72c2c18efc2d4bdb93596523d5e#egg=codejail
+-e git+https://github.com/edx/codejail.git@17b45de7191f561e9a2c1b12c89489a8ba8e0758#egg=codejail
```

Comment:
Any reason for not using the release tag instead of the commit hash?

Reply:
Generally we want to use the hash even if it is tagged since tags can move.